website/content/changelog/v1.2.1.mdx

```diff
@@ -20,7 +20,7 @@ version: 1.2.1
   - Add minor bump for adding subcommand `tokens` to generate Theme Typings
 
     See
-    [https://chakra-ui.com/docs/theming/advanced](https://chakra-ui.com/docs/theming/advanced)
+    [https://v2.chakra-ui.com/docs/theming/advanced](https://v2.chakra-ui.com/docs/theming/advanced)
```

Comment:
Same for here as well

Reply:
I changed all the links that were broken and pointed it to the v2 working link. I would say if the link works either way it shouldn't be changed, but I think the broken links should be changed. I'm open to your reasoning though.